bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
```

Comment:
now I see the problem, please delete all of these files and ignore the bundle dir. This should not have been committed.

Reply:
Removed.